postgres/requirements.in

```diff
@@ -1,2 +1,3 @@
 psycopg2-binary==2.8.4
+mmh3==2.5.1
```

Comment:
```suggestion
```

Already included for `datadog_checks_base` and there doesn't seem to be a unique requirement for it in this check anyway.